src/utils/pathsDefinition.ts

```diff
@@ -0,0 +1,168 @@
+import PathDTO from "../dtos/PathDTO";
```

Comment:
> Isso são os caminhos possíveis da instância do problema?

Se for bota a foto da instância no comentário, a foto do grafo

Reply:
Exato, irei botar